src/gui/sdl_mapper.cpp

```diff
@@ -3246,21 +3246,37 @@ static void MAPPER_Destroy(Section *sec) {
 	SDL_QuitSubSystem(SDL_INIT_JOYSTICK);
 }
 
-void MAPPER_BindKeys(Section *sec)
+static bool should_skip_bind_keys(const Section_prop* section)
```

Comment:
Rename to `should_skip_unchanged_window_titlebar`?

Reply:
Renamed to `curr_` / `prev_`.

Reply:
Renamed to `should_skip_unchanged_titlebar`.